nemo/collections/asr/parts/submodules/jasper.py

```diff
@@ -481,13 +485,13 @@ def forward_for_export(self, x, lengths):
             y = self.fc(y)  # [B, 1, C]
             y = y.transpose(1, -1)  # [B, C, 1]
 
-        # Note: Keep for future, in case we improve WER from doing so.
-        # if self.context_window >= 0:
-        #     y = F.interpolate(y, size=x.shape[-1], mode=self.interpolation_mode)
+            # Note: Keep for future, in case we improve WER from doing so.
+            # if self.context_window >= 0:
+            #     y = F.interpolate(y, size=x.shape[-1], mode=self.interpolation_mode)
 
-        y = torch.sigmoid(y)
-        y = x * y
-        return y, lengths
+            y = torch.sigmoid(y)
+            y = x * y
+        return y.to(dtype=dtype), lengths
```

Comment:
The output of SE block must be fp32. We see some instability in fp16 training otherwise.

Reply:
done.